concepts.md

```diff
@@ -1,5 +1,25 @@
-# Concepts
+# Concepts: Hypermedia and REST
 
-### Introduction
+## Introduction
 
 At the heart of Xing is a Hypermedia JSON format for sending and receiving data between a browser client and an API server. This is the basic building block upon which we've built Xing's Ruby on Rails backend and its AngularJS frontend. This chapter introduces the core concepts and structure of our data format.
+
+A full discussion of Hypermedia and REST are far beyond the scope of this document, but we provide a brief introduction here and links to further reading below.
+
+## Hypermedia
+
+At its core, Hypermedia is about decoupling client and server code from dependence on rigid URL structure: this sort of coupling leads to difficult API versioning and all sorts of development difficulty down the road.  Instead, in a hypermedia application front and back-end code need only agree on the contents and structure of data. To find resources, a hypermedia application works the same way humans do when surfing the web: it follows URL links. Because the client application follows links rather than visits hardcoded URLs, the back-end can restructure the API to a large extent, independently of the frontend. 
+
+In a Xing application, a single HTTP API resource called /resources lists the top-level resources available to a front-end client.  The client "surfs" those links to access resources. Individual resources themselves link to related resources.  Related resources  can be separate (and linked to), or embedded inside larger resources (to save requests), and in many cases a properly structured Xing frontend will work either way.  The back-end can make these optimization decisions at any time ... even on the fly!
+
+If you're familiar with the acronym HATEOAS (hypermedia as the engine of application state), Xing makes every attempt to be a true HATEOAS-driven web development platform.
```

Comment:
I'd just leave this paragraph out-- it's too much of an offhand remark talking directly to the reader. If you're familiar with HATEOS... you actually just don't need to read this intro.